identity/aziot-identityd/src/lib.rs

```diff
@@ -51,7 +51,7 @@ macro_rules! match_id_type {
     };
 }
 
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
Derive `Copy` too because all the variants are empty, and then you won't need to `.clone()` it explicitly.

Reply:
Fixed.